src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n+4) {
+    margin-bottom: 0;
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $card-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $card-second-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review-count {
+    font-size: 10px;
+    font-weight: 400;
+    color: $card-color;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $card-second-color;
+  }
+
+  &__price-amount {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $card-color;
+  }
+
+  &__button-link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    font-family: inherit;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    background-color: $acdc-color;
+    color: white;
+    text-transform: uppercase;
+    text-align: center;
+
+    transition: color 1s, background-color 1s;
+  }
+
+  &:hover &__button-link {
+    background-color: #fff;
+    color: $acdc-color;
+    border-radius: 5px;
+    border-color: $acdc-color;
+    transition: color 0.5s, background-color 0.5s;
+  }
```

Comment:
i think you have some problem with button-link, the button reacts when I hover over any element of the card, not just the button

Reply:
It was done on purpose

Reply:
it was made on purpose